en/flight_modes/manual_stabilized.md

```diff
@@ -0,0 +1,21 @@
+# Manual/Stabilized Flight Mode (Multicopter)

```

Comment:
FYI This is a new stand along doc for Manual MC mode. This is referred to as "Manual/Stabilized" across our docset. In QGC it is either Manual or Stabilized - selecting either gives you the same mode.